### PR TITLE
More string.Format --> string interpolation

### PR DESCRIPTION
**Pull request by SeanKilleen**
## Summary

Move some of the code samples from `string.Format` to string interpolation.

Supports https://github.com/dotnet/docs/issues/9002

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 1]
```diff
@@ -477,7 +477,7 @@ The steps in this section add properties and helper methods to the form class th
                 UpdateStatus("Workflow Canceled.")
             Else
                 Dim Turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
                Dim turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 2]
```diff
@@ -477,7 +477,7 @@ The steps in this section add properties and helper methods to the form class th
...
-                UpdateStatus(String.Format("Congratulations, you guessed the number in {0} turns.", Turns))
+                UpdateStatus($"Congratulations, you guessed the number in {Turns} turns.")
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
                UpdateStatus($"Congratulations, you guessed the number in {turns} turns.")
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
Small change unrelated to your changes. Local variables should be `camelCase`d

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 3]
```diff
@@ -499,7 +497,7 @@ The steps in this section add properties and helper methods to the form class th
         else
         {
             int Turns = Convert.ToInt32(e.Outputs["Turns"]);
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
            int turns = Convert.ToInt32(e.Outputs["Turns"]);
```

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 4]
```diff
@@ -499,7 +497,7 @@ The steps in this section add properties and helper methods to the form class th
...
-            UpdateStatus(string.Format("Congratulations, you guessed the number in {0} turns.", Turns));
+            UpdateStatus($"Congratulations, you guessed the number in {Turns} turns.");
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
            UpdateStatus($"Congratulations, you guessed the number in {turns} turns.");
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 5]
```diff
@@ -510,16 +508,12 @@ The steps in this section add properties and helper methods to the form class th
     ```vb
     wfApp.Aborted = _
         Sub(e As WorkflowApplicationAbortedEventArgs)
-            UpdateStatus(String.Format("Workflow Aborted. Exception: {0}" & vbCrLf & "{1}", _
-                e.Reason.GetType().FullName, _
-                e.Reason.Message))
+            UpdateStatus($"Workflow Aborted. Exception: {0e.Reason.GetType().FullName}" & vbCrLf & $"{e.Reason.Message}")
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
            UpdateStatus($"Workflow Aborted. Exception: {0e.Reason.GetType().FullName}{Environment.NewLine}{e.Reason.Message}")
```

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 6]
```diff
@@ -510,16 +508,12 @@ The steps in this section add properties and helper methods to the form class th
...
         End Sub
 
     wfApp.OnUnhandledException = _
         Function(e As WorkflowApplicationUnhandledExceptionEventArgs)
-            UpdateStatus(String.Format("Unhandled Exception: {0}" & vbCrLf & "{1}", _
-                e.UnhandledException.GetType().FullName, _
-                e.UnhandledException.Message))
+            UpdateStatus($"Unhandled Exception: {e.UnhandledException.GetType().FullName}" & vbCrLf & $"{e.UnhandledException.Message}")
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
            UpdateStatus($"Unhandled Exception: {e.UnhandledException.GetType().FullName}{Environment.NewLine}{e.UnhandledException.Message}")
```

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 7]
```diff
@@ -594,23 +584,19 @@ The steps in this section add properties and helper methods to the form class th
                     UpdateStatus("Workflow Canceled.")
                 Else
                     Dim Turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
                    Dim turns As Integer = Convert.ToInt32(e.Outputs("Turns"))
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md (hunk 1 above)
Comment (SeanKilleen) [reply docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]: @Youssef1313 I don't disagree, but I'd like to keep the scope of these changes to formatting specifically so as not to muddy the PR. I think camel-casing local variables is a fine PR idea; I just don't want to mix-and-match here.

**Comment by SeanKilleen** [comment]
knocked a few of these out -- will have to stop there for now. :) 

This is ready for review.

**Comment by SeanKilleen** [comment]
@Youssef1313 I don't disagree with your observation on local variables, but I'd like to keep the scope of these changes to formatting specifically, so as not to muddy the PR. 

I think camel-casing local variables is a fine PR idea (a good regex search would probably un-earth a great PR in fact!) I just don't want to mix-and-match here. 

Given that, would you be OK with me marking your comments as resolved?

**Comment by Youssef1313** [comment]
@SeanKilleen, Well, while reviewing I thought there would be a 1 or 2 of PascalCase local variables. But as the number is a bit higher, I'm agreeing with you that it's better to do that in another PR.
There are other suggestions regarding the new line addition inside of the interpolated strings, which I think are suitable to be done in this PR.

(I can't find the `Mark as resolved` button).

**Review by BillWagner — APPROVED** [review]
Thanks for making these changes @SeanKilleen 

I'm glad you and @Youssef1313 could work together and come to agreement on the other changes.

I've resolved the remaining conversations, and this is ready to :shipit: now.

I'll merge this, and you should see the changes on the live site in the next day or two. Thanks for making your first contribution to docs! 🎆

**Pull request merged**